lib/twitter_cldr/js/mustache/numbers/rbnf/rule_parser.coffee

```diff
@@ -0,0 +1,69 @@
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+class TwitterCldr.RBNFRuleParser extends TwitterCldr.Parser
+  do_parse : (options = {}) ->
+    @switch([])
+
+  switch : (list) -> #TODO - This name was "switch". Figure it out.
```

Comment:
it got leftover somehow. I'll do a passthrough removing all TODOs before the PR is merged